lib/AST/ES6Class.cpp

```diff
@@ -0,0 +1,675 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include "hermes/AST/ES6Class.h"
+#include "hermes/AST/RecursiveVisitor.h"
+#include "hermes/Parser/JSLexer.h"
+
+namespace hermes {
+struct ClassMemberKey {
```

Comment:
If this class (and similar types) is intended to be used only here, perhaps it should be in an anonymous namespace?